src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/plugin/core_threading_tests.cpp

```diff
@@ -3,14 +3,22 @@
 //
 
 #include <behavior/plugin/core_threading.hpp>
+#ifdef __GLIBC__
+#include <gnu/libc-version.h>
+#if __GLIBC_MINOR__  >= 34
+    #define ENABLETESTMULTI
+#endif
+#endif
```

Comment:
should not it be disabled for other devices as well? Maybe we need to add skip in the test body itself?

Reply:
Hi ilya, the individual device will not call getAvailableDevice in their plugin, just skip the test for multi/auto is enough. I'll test this PR for several cycles to see how the result goes.

Reply:
Originally we had getAvailableDevices multi-threading test for all the devices here https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/include/behavior/plugin/core_threading.hpp#L240-L252
can we add such test with the guard you have? In this case such case will be run on all devices. Thanks

Reply:
Hi ilya, the link above is test for LoadNetwork, I suppose you meant to refer to below test in ieFuncTest?
https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/inference_engine/core_threading.cpp#L134

Reply:
yes,  I'll enable that in the following PR.

Reply:
@songbell  CVS-93003